Dockerfile

```
@@ -1,16 +1,22 @@
-# Node Base Image
-FROM node:12.2.0-alpine
+# Use specific Node.js version
+FROM node:latest 
```

Hadolint (2.12.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)